tests/benchmarks/sys_kernel/testcase.yaml

```diff
@@ -1,5 +1,6 @@
 tests:
   benchmark.kernel:
     arch_exclude: nios2 riscv32 xtensa
+    platform_exclude: qemu_cortex_m3
```

Comment:
the prj.conf streches the time between timer interrupts. This on qemu will overflow the counter.
And what is the point in running a benchmark on an emulation platform?

Reply:
>Why is this not a problem on any of our other emulated platforms?

In arm we have only 24 bits to represent the count value. This maybe one reason. The other the clock speed on other platforms might be lesser. So it ends up streaching the timer Interrupts according to the prj.conf.

>Because the test still reports success or failure, and sanitycheck only runs on emulated platforms. If there is some functional issue that causes this test to crash we need to know about it.

As mentioned above, this is to do with the timer interrupts. One workaround would be to create a new prj.conf for qemu_cortex_m3.